src/main/java/org/terasology/lavalandgenerator/WorldGenerator.java

```diff
@@ -0,0 +1,30 @@
+package org.terasology.lavalandgenerator;
+import build.classes.org.terasology.core.world.generator.facetProviders.SeaLevelProvider;

```

Comment:
This line is throwing errors on my system, since it required you to have compiled SeaLevelProvider into a jar before compiling your module. That's a bad practice in general. You should change this to
`import org.terasology.core.world.generator.facetProviders.SeaLevelProvider;` instead

Reply:
Ok, it's fixed now. Not sure how that happened; it doesn't have build.classes on the original on my computer.